tgui/build.sh

```diff
@@ -0,0 +1,11 @@
+#RUN THIS IN THE tgui/ folder
+#!/bin/bash
+set -e
+export NODE_VERSION=4
+
+if [ ! -d "/tmp/nvm" ]; then
+	rm -rf /tmp/nvm && git clone https://github.com/creationix/nvm.git /tmp/nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source /tmp/nvm/nvm.sh && nvm install $NODE_VERSION
```

Comment:
Not thrilled with this script but oh well, supporting linux development is a good idea